ch04/ex-scripts.md

```diff
@@ -65,3 +65,13 @@ The script should end only with a complete end script tag.
        <head>
          <script>
            ' "</scri" "pt>" '
+
+
+ Test `<script>` tags with attributes
+
+    >>> test_parse("<script defer src='my-script.js'>Should be treated as text</script>")
```

Comment:
Change the contents of the `<script>` tag to be `a<b`. This will test that the browser actually enters "script tag" mode where `<` and `>` are handled specially.